tests/test_SpellingResolver.py

```diff
@@ -1,19 +1,28 @@
-from SpellingResolver import test_spelling_resolver
+import unittest, json
+import azure.functions as func
+from ddt import ddt, data, unpack
+from SpellingResolver.main import main
 
-mm = [
-    'anton marta 123',
-    'siegfried dora 2 * 7',
-    'toni berta 22',
-    'd e 3 times 7 2 times 3'
-]
+@ddt
+class TestSpellingResolver(unittest.TestCase):
+    @data(  (dict(query='anton marta 123', locale='de'), 'am123'),
+            (dict(query='siegfried dora 2 * 7', locale='de'), 'sd77'),
+            (dict(query='toni berta 22', locale='de'), 'tb22'),
+            (dict(query='d e 3 times 7 2 times 3', locale='de'), 'de77733'))
```

Comment:
I think that should be change to 
@data(  (dict(query='anton marta 123', locale='de'), {
                "original": "anton marta 123",
                "resolved": "a m 123",
                "resolved_nospace": "am123",
                "first_letters": "a m 123",
                "first_letters_nospace": "am123"
            }),
            (dict(query='siegfried dora 2 * 7', locale='de'), {
                "original": "siegfried dora 2 * 7",
                "resolved": "s d 77",
                "resolved_nospace": "sd77",
                "first_letters": "s d 77",
                "first_letters_nospace": "sd77"
            }),
            (dict(query='toni berta 22', locale='de'), {
                "original": "toni berta 22",
                "resolved": "toni b 22",
                "resolved_nospace": "tonib22",
                "first_letters": "t b 22",
                "first_letters_nospace": "tb22"
            }),
            (dict(query='d e 3 times 7 2 times 3', locale='de'), {
                "original": "d e 3 times 7 2 times 3",
                "resolved": "d e 777 33",
                "resolved_nospace": "de77733",
                "first_letters": "d e 777 33",
                "first_letters_nospace": "de77733"
            }))